Makefile

```diff
@@ -283,7 +283,7 @@ lint_analytics_events: .yardoc ## Checks that all methods on AnalyticsEvents are
 
 lint_analytics_events_sorted:
 	@test "$(shell grep '^  def ' app/services/analytics_events.rb)" = "$(shell grep '^  def ' app/services/analytics_events.rb | sort)" \
-		|| (echo 'Error: methods in analytics_events.rb are not sorted alphabetically' && exit 1)
+		|| (echo -e '\033[1;31mError: methods in analytics_events.rb are not sorted alphabetically\033[0m' && exit 1)
```

Comment:
The `-e` here appears to be output literally in the build output?

![image](https://github.com/18F/identity-idp/assets/1779930/52911e2d-d38f-4723-a028-409a88b758d1)

Reply:
But StackOverflow said to use it! :trollface: ;)

I had noticed this locally, and chalked it up to a Mac vs. coreutils issue or something. I wonder if it's actually bash vs. zsh and it's a built-in?

Since it seems extraneous either way I'll remove it!

Are you generally on board with the idea of this PR?

Reply:
Part of the reason I thought to look is my own history with common commands having different support across OS, so I suspected maybe that's why it wasn't working as expected but could be something that a StackOverflow describes.

>Are you generally on board with the idea of this PR?

I think it makes sense if it helps boost visibility of the issue. My main concerns would be (a) applying this ad-hoc rather than across the board, and (b) the color codes hurt the readability of the code. But overall I feel it could be a net-positive 👍

Reply:
> My main concerns would be (a) applying this ad-hoc rather than across the board, and (b) the color codes hurt the readability of the code. But overall I feel it could be a net-positive 👍

Thanks! I am trying to keep the scope on this as small as I can because I don't have a ton of spare cycles, but I agree with both of these. I was tempted to implement a little `echo_red` function or something, but it felt a little weird to put that in a Makefile. And making the other `echo`s for errors red should really result in me testing triggering them.

I'll take a screenshot of this without the `-e`, then back out the change to AnalyticsEvents that was introduced to trigger this, and then merge. If someone is feeling inspired down the road we can extend it.